### DISCUSSION TITLE
Ability to dismiss blocked user notification

### DISCUSSION THREAD

**Discussion by Kowlin** [community/community#5152] [Mobile] [answered]
Sometimes blocking other users on GitHub is inevitable, those users may have contributed in the past. While I understand that the notification for having that is sometimes an handy way to judge if you should use/contribute the repository. However, it doesn't really help when you're a maintainer of the repository.

This feature natively exists on desktop, and doesn't appear on mobile, nor does it respect the setting. And for that reason I'd like to request the option to dismiss the banner.

![image](https://user-images.githubusercontent.com/10947836/129768759-4f4cc03b-2b99-4544-ac9b-aee6b4478f96.png)

![photo_2021-08-17_18-56-11](https://user-images.githubusercontent.com/10947836/129768610-bfc090d4-9127-4520-94f2-4d94f2b06174.jpg)

**Comment by Iron-Ham** [comment, marked as answer]
That's great feedback. I've opened up an internal issue to track this. Thank you for reporting this!

**Reply by Cremalsa** [reply]
1

**Reply by Cremalsa** [reply]
Open
